File: collect-stats.pl
#
# inc-nums.cpp
# Copyright (C) 2019 Shlomi Fish <shlomif@cpan.org>
#
# Distributed under terms of the MIT license.
#

use strict;
use warnings;

my $S = 0;
my $J = 0;
while (<>)
{
    ++( substr( $_, 0, 1 ) eq 'S' ? $S : $J );
}
print "S\t$S\nInt\t$J\n";

File: collect-stats.py
#
# inc-nums.cpp
# Copyright (C) 2019 Shlomi Fish <shlomif@cpan.org>
#
# Distributed under terms of the MIT license.
#

import sys
with open(sys.argv[1], "r") as f:
    S = J = 0
    for line in f:
        if line[0] == 'S':
            S += 1
        else:
            J += 1
    print("S\t{}\nInt\t{}".format(S, J))
